api-specs/api_spec_PGS.yaml

```diff
@@ -575,6 +575,35 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/XPayRefundResponse500'
+  /request-payments/creditCard/{requestId}:
+    get:
+      summary: retrieve CreditCard payment request
+      tags:
+        - Credit Card
+      parameters:
+        - in: path
+          required: true
+          name: requestId
+          description: Id of the request
+          example: "41bc2409-5926-4aa9-afcc-797c7054e467"
+          schema:
+            type: string
+        - in: header
+          name: MDC-Fields
+          description: MDC information
+          example: 97g10t83x7bb0437bbc50sdf58e970gt
+          schema:
+            type: string
+          required: false
+      responses:
+        "200":
+          description: OK
+          content:
+            '*/*':
+              schema:
+                $ref: '#/components/schemas/CcPaymentInfoResponse'
+        "404":
+          "description": Not Found
```

Comment:
This doesn't specify a schema for the 404 response. IIRC it should be something like
```json
{
  "error": "RequestId non trovato"
} 
```